app/src/main/java/org/fossasia/openevent/events/ErrorDialogEvent.java

```diff
@@ -0,0 +1,59 @@
+package org.fossasia.openevent.events;
+
+import android.util.Log;
+
+import org.fossasia.openevent.OpenEventApp;
+import org.fossasia.openevent.api.APIClient;
+
+import retrofit.RetrofitError;
+
+/**
+ * Created by shivenmian on 28/12/15.
+ */
+public class ErrorDialogEvent {
+
+    public String ErrorType;
+    public String ErrorDesc;
+    public static final String TYPE = "RetrofitError Type";
+    public static final String ERROR_CODE = "Error Code";
+
+    public ErrorDialogEvent(String s1, String s2){
+        this.ErrorType = s1;
+        this.ErrorDesc = s2;
+    }
+
+    public void ErrorHandlerFunction(RetrofitError cause, ErrorDialogEvent event){
+
+        if (cause.getKind().equals(RetrofitError.Kind.CONVERSION)) {
```

Comment:
Yeah, I know. I just mean to say that it gives the wrong dialog. It's supposed to give an error.

http://www.mocky.io/v2/5682128c120000993893a2c1

This is a malformed JSON. This should give a Conversion Error instead of an HTTP 200.